subsys/fw_info/fw_info.c

```diff
@@ -154,8 +155,8 @@ SYS_INIT(check_ext_api_requests, POST_KERNEL, CONFIG_APPLICATION_INIT_PRIORITY);
 /** Value to write to the "valid" member of fw_info to invalidate the image. */
 #define INVALID_VAL 0xFFFF0000
 
-BUILD_ASSERT_MSG((INVALID_VAL & CONFIG_FW_INFO_VALID_VAL) \
-			!= CONFIG_FW_INFO_VALID_VAL, \
+BUILD_ASSERT_MSG((INVALID_VAL & CONFIG_FW_INFO_VALID_VAL)
+			!= CONFIG_FW_INFO_VALID_VAL,
```

Comment:
`!=` should land in the first line

Reply:
Can I ask why? I find it more readable this way.